packages/main/src/plugin/cli-tool-registry.ts

```diff
@@ -137,4 +145,40 @@ export class CliToolRegistry {
       };
     });
   }
+
+  getCliTool(id: string): CliToolInfoApi | undefined {
+    const cliTool = this.cliTools.get(id);
+    if (!cliTool) {
+      return undefined;
+    }
+    return {
+      id: cliTool.id,
+      name: cliTool.name,
+      displayName: cliTool.displayName,
+      markdownDescription: cliTool.markdownDescription,
+      state: cliTool.state,
+      images: cliTool.images,
+      extensionInfo: {
+        id: cliTool.extensionInfo.id,
+        label: cliTool.extensionInfo.label,
+      },
+    };
+  }
+
+  getCliTools(): CliToolInfoApi[] {
+    return Array.from(this.cliTools.values()).map(cliTool => {
+      return {
+        id: cliTool.id,
```

Comment:
maybe a toInfo() like method shared between getCliTool and getCliTools (transforming a CliToo to CliTooInfo ?)

Reply:
yes, better. done